src/store/index.js

```diff
@@ -47,18 +65,41 @@ export default createStore({
       }
     },
 
+    async getPolls({ commit }) {
+      try {
+        let pageNo = 1;
+        const response = await api.get(`poll/list/${pageNo}?limit=4`,);
```

Comment:
pageNo and limit should be comes from component or composition API, not in store file @ShivanshSinghnnovative